android/src/main/java/com/pdftron/reactnative/views/DocumentView.java

```diff
@@ -468,6 +468,10 @@ public void setCurrentUserName(String currentUserName) {
         mCurrentUserName = currentUserName;
     }
 
+    public void setHideReplyReviewState(boolean hideReplyReviewState) {
```

Comment:
Follow naming convention of other methods. Use `setReplyReviewStateEnabled`. Will also need to update documentation and code below.